packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Dependencies/Dependency/index.tsx

```diff
@@ -185,9 +189,14 @@ export const Dependency = ({
           {versions.length === 0 ? null : (
             <Select
               css={{ width: '80px' }}
-              value={versions.find(v => v === dependencies[dependency])}
+              value={versionFromDropdown}
```

Comment:
Since the version from dropdown might be undefined, should the value be version from dropdown or the defined version that's added to the options?

Reply:
Oh good point, that's more explicit. I'll update this!

Reply:
Updated!